src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Byte128Vector.java

```diff
@@ -708,22 +708,32 @@ Byte128Mask xor(VectorMask<Byte> mask) {
         @Override
         @ForceInline
         public int trueCount() {
-            return VectorSupport.maskReductionCoerced(VECTOR_OP_MASK_TRUECOUNT, Byte128Mask.class, byte.class, VLENGTH, this,
-                                                      (m) -> trueCountHelper(((Byte128Mask)m).getBits()));
+            return (int) VectorSupport.maskReductionCoerced(VECTOR_OP_MASK_TRUECOUNT, Byte128Mask.class, byte.class, VLENGTH, this,
+                                                      (m) -> (long)trueCountHelper(((Byte128Mask)m).getBits()));
```

Comment:
```suggestion
            return (int) VectorSupport.maskReductionCoerced(VECTOR_OP_MASK_TRUECOUNT, Byte128Mask.class, byte.class, VLENGTH, this,
                                                      (m) -> trueCountHelper(m.getBits()));
```

We don't need the cast to `long` for the return from the lambda expression. And I think the cast of the mask is redundant too. Same applies to the other three methods.